Dockerfile

```diff
@@ -4,4 +4,16 @@ RUN mkdir /app
 COPY ./target/eth-connector-0.0.1-SNAPSHOT.jar /app
```

Comment:
[nitpick] : version will be dynamic in nature.

Reply:
Agree, we need an automated way to increment the version. Will add this to the backlog to pick this up later.